ui/apps/platform/src/Containers/PolicyCategories/PolicyCategoriesList.tsx

```diff
@@ -19,15 +19,15 @@ function PolicyCategoriesList({
     return (
         <SimpleList onSelect={() => {}}>
             {policyCategories.map((category) => {
-                const { id, name } = category;
+                const { id, name, isDefault } = category;
                 return (
                     <SimpleListItem
                         key={id}
                         onClick={() => {
                             setSelectedCategory(category);
                         }}
                         isActive={false}
```

Comment:
Will `isActive` prop have a conditional value in a future pull request?

Reply:
oh no it's residual from styling i thought i had to do. good eyes, i'll remove it